docs/expressions.md

```diff
@@ -0,0 +1,89 @@
+# Expressions Query Language (QL)
+Solaris allows to specify filters for selecting Logs and their records in form of boolean exression. An expression is a text which consists contains constants, arguments and operations that form the request for selecting logs and their record. We will try to explain it concisely here so that the reader can apply it immediately.
```

Comment:
Solaris allows you to specify filters for selecting logs and their records in the form of Boolean expressions. An expression is a text that contains constants, arguments, and operations that form the query for selecting logs and their records. We will try to explain it briefly here so that the reader can apply it immediately.

Reply:
Done